### ISSUE TITLE
Use subjects rather than patients throughout

### ISSUE THREAD

**Issue by kylebaron** [metrumresearchgroup/bbr#317]
# Summary
As a user, I want bbr to refer to individuals in an analysis as "subjects" rather than "patients".

# Tests
- tests/testthat/test-print.R
   - print.bbi_nonmem_summary works basic FOCE model

**Comment by seth127** [comment]
This would actually be a change in `bbi`. I've implemented it here: https://github.com/metrumresearchgroup/bbi/pull/209

When that merges and is released as `bbi 3.0.2` I'll do a little PR in `bbr` to reconcile anything. It looks like it's mostly in test references, which are easy to re-generate:

```
~/bbr$ grep -rni patients .
./R/model-summary.R:19:#'    method, numbers of patients and records, significant digits, run time, and./R/summary-log.R:24:#' * `number_of_patients` -- Count of unique patients in the input data set, extracted from `$run_details`.
./R/print.R:163:  cat_line(glue("Records: {.d$number_of_data_records}\t Observations: {.d$number_of_obs}\t Patients: {.d$number_of_patients}\n\n"))
./R/aaa.R:154:  "number_of_patients",./vignettes/using-summary-log.Rmd:114:    number_of_patients,
./inst/test-refs/1_summary_obj.R:9:    data_set = "../../../../extdata/acop.csv", number_of_patients = 40L,
./inst/test-refs/print-refs/print_nmsum_iovmm.txt:3:Records: 13500       Observations: 12600     Patients: 300
./inst/test-refs/print-refs/print_nmsum_example2_saemimp_noargs.txt:3:Records: 2400      Observations: 2000      Patients: 400
./inst/test-refs/print-refs/print_nmsum_acop-iov_fixedTRUE_nrow15.txt:3:Records: 7566    Observations: 6396      Patients: 39
./inst/test-refs/print-refs/print_nmsum_acop-iov_noargs.txt:3:Records: 7566      Observations: 6396      Patients: 39
./inst/test-refs/print-refs/print_nmsum_acop-iov_fixedTRUE.txt:3:Records: 7566   Observations: 6396      Patients: 39
./inst/test-refs/print-refs/print_nmsum_1001.txt:3:Records: 921  Observations: 599       Patients: 240
./inst/test-refs/print-refs/print_nmsum_example2_saemimp_fixedTRUE_offdiagTRUE.txt:3:Records: 2400       Observations: 2000      Patients: 400
./inst/test-refs/print-refs/print_nmsum_example2_saemimp_offdiagTRUE.txt:3:Records: 2400         Observations: 2000      Patients: 400
./inst/test-refs/print-refs/print_nmsum_1.txt:3:Records: 799     Observations: 760       Patients: 40
./inst/test-refs/print-refs/print_nmsum_example2_saemimp_fixedTRUE.txt:3:Records: 2400   Observations: 2000      Patients: 400
```

**Comment by seth127** [comment]
Closed by #320
